server/src/main/java/org/opensearch/index/IndexSettings.java

```diff
@@ -578,6 +579,14 @@ public final class IndexSettings {
         Property.InternalIndex
     );
 
+    public static final Setting<String> DEFAULT_SEARCH_PIPELINE = new Setting<>(
+        "index.default_search_pipeline",
```

Comment:
It seems like other search related setting (INDEX_SEARCH_IDLE_AFTER, INDEX_SEARCH_THROTTLED) use `index.search.xxx` convention, may be:

```suggestion
        "index.search.default_pipeline",
```

(since we have DEFAULT_PIPELINE already for ingestion)